docs/db-cloud-shared/tutorials/monitor-host-metrics/go-demo.md

```diff
@@ -0,0 +1,124 @@
+
+Now we will create a quick start demo step by step to collect host metrics and send them to Greptime. The demo is based on [OTLP/HTTP](https://opentelemetry.io/). For reference, you can view the complete demo on [Github](https://github.com/GreptimeCloudStarters/quick-start-go).
+
+To begin, create a new directory named `quick-start-go` to house our project. Then, run the command `go mod init quick-start` in the directory from your terminal. This will create a `go.mod` file, which is used by Go to manage imports.
```

Comment:
```suggestion
To begin, create a new directory named `quick-start-go` to host our project. Then, run  `go mod init quick-start` in the directory from your terminal. This will create a `go.mod` file, which is used by Go to manage imports.
```